### PR TITLE
small fixes for chris rousel a11y audit

### PR DESCRIPTION
**Pull request by alexbowen**

### CODE REVIEW

**Review by cesidio — APPROVED** [review]

File: app/helpers/vacancies_helper.rb [hunk 1]
```diff
@@ -180,8 +180,8 @@ def vacancy_working_patterns(vacancy)
     return vacancy_working_patterns_summary(vacancy) unless vacancy.full_time_details? || vacancy.part_time_details?
 
     safe_join [
-      (tag.li { t("jobs.full_time_details", details: vacancy.full_time_details) } if vacancy.full_time_details?),
-      (tag.li { t("jobs.part_time_details", details: vacancy.part_time_details) } if vacancy.part_time_details?),
+      (tag.div { t("jobs.full_time_details", details: vacancy.full_time_details) } if vacancy.full_time_details?),
```
Comment (cesidio) [inline app/helpers/vacancies_helper.rb:183]: if this changes from a list element, you want to remove the wrapping `ul` element

Comment (alexbowen) [reply app/helpers/vacancies_helper.rb:183]: there is no wrapping ul element, thats the point

Comment (cesidio) [reply app/helpers/vacancies_helper.rb:183]: I see the wrapping element

https://github.com/DFE-Digital/teaching-vacancies/blob/main/app/views/publishers/vacancies/_vacancy_review_sections.html.slim#L43

Comment (alexbowen) [reply app/helpers/vacancies_helper.rb:183]: hmmm i dont, will investigate why this doesnt render properly

Screenshot 2022-09-12 at 09 50 36

**Pull request merged**